app/src/main/java/org/sopt/havit/ui/save/SaveFragment.kt

```diff
@@ -72,11 +119,16 @@ class SaveFragment(categoryName:String) : BottomSheetDialogFragment() {
             dismiss()
         }
         binding.btnSaveNext.setOnClickListener {
-            val intent = Intent(requireContext(), ShareActivity::class.java).apply {
-                putExtra("url", binding.etSaveUrl.text.toString())
+            if (isFullPath(binding.etSaveUrl.text.toString())) {
+                val intent = Intent(requireContext(), ShareActivity::class.java).apply {
+                    putExtra("url", binding.etSaveUrl.text.toString())
+                }
+                startActivity(intent)
+                dismiss()
+            } else {
+                setCustomToast()
             }
-            startActivity(intent)
-            dismiss()
+
         }
         binding.etSaveUrl.addTextChangedListener(object : TextWatcher {
```

Comment:
#252

위 이슈에서와 같이 TextWatcher를 사용하지 않고 간단하게 코딩할 수 있습니다~

또한 if 문도 아래와 같이 간단하게 나타내면 좋을것 같아요~

```suggestion
        binding.etSaveUrl.addTextChangedListener{
            val isExist=  binding.etSaveUrl.text.isNotEmpty()
            saveViewModel.setClick(isExist)
         }
```